lib/core-integration/src/Test/Integration/Scenario/API/Byron/Transactions.hs

```diff
@@ -595,3 +601,91 @@ spec = describe "BYRON_TRANSACTIONS" $ do
         r <- request @([ApiTransaction n]) ctx link Default Empty
         expectResponseCode HTTP.status404 r
         expectErrorMessage (errMsg404NoWallet $ w ^. walletId) r
+
+    -- Golden from https://github.com/input-output-hk/cardano-sl/pull/4278#issuecomment-600553878
+    it "BYRON_SCRYPT_TRANS_CREATE_01 - wallet with password" $ \ctx -> do
+        pendingWith "password in golden seems wrong"
+        -- I also tried a few tweaks to our 'preparePassphrase', but to no help.
+        -- Either the passphrase is wrong, or the implementation is wrong, or
+        -- both.
+        flip runByronScryptGolden ctx $ ByronScryptGolden
+            { mnemonic = T.words
+                "inhale arm pilot fitness ceiling october donate \
+                \between language all limit taxi"
+            , encryptedRootXPrv =
+                "f6e79f49b8999a39d7e970e42d0a91224ecacefc3aa1edb342f34eb8bc6c2f\
+                \c63e743b862b312a6f92ba0161d4d53c3ee5a2bd8085476d9575765c49dcee\
+                \cbe54b34ec47daf9b7ebc6bdb706622616451c000e85ba81c7449ae436a8cb\
+                \bf3aab98e5cc704977bd11bb0ba8d5b5571a705704cb9334d27a048532eab4\
+                \9a698c2d"
+            , passwordHash =
+                "31347c387c317c5743413633702f6a487a5777575278756756344e31685479\
+                \3470646c6d4f76665177653863775a575472784f79773d3d7c796341722f61\
+                \326f4f777a736e4e746f4e655049416e4f6b7978426549494a6b59623039574\
+                \b564a7159493d"
+            , password =
+                "0000000000000000000000000000000000000050415441544520504154415445"
```

Comment:
I think password is probably not fit for use litelarly in cardano-wallet as it's most likely old wallet's "format".
In old wallet password was:
> The spendingPassword is optional but highly recommended. It a string of 32 characters, encoded in base 16, yielding to an hexadecimal sequence of 64 bytes. This passphrase is required for sensitive operations on the wallet and adds an extra security layer to it.

https://input-output-hk.github.io/cardano-wallet-legacy/#section/Getting-Started/Creating-a-New-Wallet

Having said that:
```
$ echo "0000000000000000000000000000000000000050415441544520504154415445" | xxd -p -r
PATATE PATATE
```
_(classic)_ ;P

Reply:
THANKS! It now works!

I had to both 
-  fix the passphrase
- drop the `hashMaybe` I was suspicious of from `preparePassphrase` (and saw no trace of in cardano-sl)

This might mean we have found the culprit for https://input-output.atlassian.net/browse/ADP-1335 then 🤞